userspace/engine/json_evt.cpp

```diff
@@ -52,9 +52,261 @@ uint64_t json_event::get_ts()
 	return m_event_ts;
 }
 
-std::string json_event_filter_check::def_format(const json &j, std::string &field, std::string &idx)
+json_event_value::json_event_value()
 {
-	return json_as_string(j);
+}
+
+json_event_value::~json_event_value()
+{
+}
+
+json_event_value::json_event_value(const std::string &val)
+{
+	if(parse_as_pair_int64(m_pairval, val))
+	{
+		m_type = JT_INT64_PAIR;
+	}
+	else if (parse_as_int64(m_intval, val))
```

Comment:
According to current C++ coding convention there should not be a space after `else if`.

Could you please check your additions to be formatted accordingly to it? There are other lines that need to be formatted according to it. Thanks :)